tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/configured_python_example.py

```diff
@@ -0,0 +1,89 @@
+    "execution_engine": {
+        "class_name": "PandasExecutionEngine",
+        "azure_options": {
+            "account_url": "<YOUR_ACCOUNT_URL>",
+            "credential": "<YOUR_CREDENTIAL>",
+        },
+    },
+    "data_connectors": {
+        "configured_data_connector_name": {
+            "class_name": "ConfiguredAssetAzureDataConnector",
+            "azure_options": {
+                "account_url": "<YOUR_ACCOUNT_URL>",
+                "credential": "<YOUR_CREDENTIAL>",
+            },
+            "container": "superconductive-public",
+            "name_starts_with": "data/taxi_yellow_trip_data_samples/",
+            "assets": {"taxi_data": None},
+            "default_regex": {
+                "pattern": "data/taxi_yellow_trip_data_samples/yellow_trip_data_sample_(\\d{4})-(\\d{2})\\.csv",
+                "group_names": ["year", "month"],
+            },
+        },
+    },
+}
+
+# Please note this override is only to provide good UX for docs and tests.
+# In normal usage you'd set your path directly in the yaml above.
+datasource_config["execution_engine"]["azure_options"][
+    "account_url"
+] = "superconductivetests.blob.core.windows.net"
+datasource_config["execution_engine"]["azure_options"]["credential"] = CREDENTIAL
+datasource_config["data_connectors"]["configured_data_connector_name"]["azure_options"][
+    "account_url"
+] = "superconductivetests.blob.core.windows.net"
+datasource_config["data_connectors"]["configured_data_connector_name"]["azure_options"][
+    "credential"
+] = CREDENTIAL
+datasource_config["data_connectors"]["configured_data_connector_name"][
+    "container"
+] = "superconductive-public"
+datasource_config["data_connectors"]["configured_data_connector_name"][
+    "name_starts_with"
+] = "data/taxi_yellow_trip_data_samples/"
+
+context.test_yaml_config(yaml.dump(datasource_config))
+
+context.add_datasource(**datasource_config)
+
+# Here is a BatchRequest naming a data_asset
+batch_request = BatchRequest(
+    datasource_name="my_azure_datasource",
+    data_connector_name="configured_data_connector_name",
+    data_asset_name="<YOUR_DATA_ASSET_NAME>",
+)
+
+# Please note this override is only to provide good UX for docs and tests.
+# In normal usage you'd set your data asset name directly in the BatchRequest above.
+batch_request.data_asset_name = "taxi_data"
+
+context.create_expectation_suite(
+    expectation_suite_name="test_suite", overwrite_existing=True
+)
+validator = context.get_validator(
+    batch_request=batch_request, expectation_suite_name="test_suite"
+)
+print(validator.head())
+
+# NOTE: The following code is only for testing and can be ignored by users.
+assert isinstance(validator, ge.validator.validator.Validator)
+assert [ds["name"] for ds in context.list_datasources()] == ["my_azure_datasource"]
+assert set(
+    context.get_available_data_asset_names()["my_azure_datasource"][
+        "configured_data_connector_name"
+    ]
+) == {"taxi_data"}
```

Comment:
Doesn't feel as robust as the check for `Inferred & Runtime` but it does still check for basic functionality. Thoughts on possibly making this more robust?

Reply:
Great point. One of our users had suggested ending the E2E test scripts, not with a Validator, but a Checkpoint. This would be a more direct translation to how we encourage users to Validate data in GE (this would be small a refactor for all of the E2E scripts, so not something we would want to ask for here)

Regarding robustness, we currently (purposely) aren't placing much emphasis on the robustness of these Docs tests.  The place for more 'complete' tests would be in the **actual** integration tests and/or unittests.